test/functional/mempool_package_limits.py

```diff
@@ -26,7 +26,7 @@ def func_wrapper(self, *args, **kwargs):
         testres_error_expected = node.testmempoolaccept(rawtxs=package_hex)
         assert_equal(len(testres_error_expected), len(package_hex))
         for txres in testres_error_expected:
-            assert "package-mempool-limits" in txres["package-error"]
+            assert "too-large-cluster" in txres["package-error"]
```

Comment:
there's a single lasting reference to "package-mempool-limits" in the comment in this file